### PR TITLE
Separate logic&add hotel

### PR DESCRIPTION
**Pull request by MaltsevaKsenia**
Add dropdown list in add tour form with list of existing hotels. 
Delete fields for country and city.
Add separate page with simple form to add a new hotel in db.
delete testadmin we don't need it anymore

Make add tour form more user friendly

### CODE REVIEW

File: src/main/java/com/epam/controller/HotelController.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                                 @RequestParam(name = "country") String country,
+                                 @RequestParam(name = "city") String city,
+                                 @RequestParam(name = "stars") Integer stars,
+                                 ModelAndView modelAndView,
+                                 RedirectAttributes redirectAttributes) {
+        redirectAttributes.addFlashAttribute("add_status", "successful");
+        Pattern pattern = Pattern.compile("^[a-zA-Z ]*$");
+        Matcher countryMatcher = pattern.matcher(country);
+        Matcher cityMatcher = pattern.matcher(city);
+        if (name.equals("")) {
```
Comment (ghost) [inline src/main/java/com/epam/controller/HotelController.java]: seems like switch

File: src/main/java/com/epam/controller/ToursController.java [hunk 2]
```diff
@@ -37,37 +46,38 @@ public ModelAndView searchTours(@RequestParam String country, @RequestParam Stri
         ModelAndView toursModel = new ModelAndView();
         toursModel.setViewName("homepage");
         try {
-            LocalDate addStartDate = Validator.getDate(startDate,true);
-            LocalDate addEndDate = Validator.getDate(endDate,true);
-            toursModel.addObject("list",toursOfferService.searchTours(null,addStartDate, addEndDate));
+            LocalDate addStartDate = Validator.getDate(startDate, true);
```
Comment (ghost) [inline src/main/java/com/epam/controller/ToursController.java]: I see that your Inellij want to fix all our code style errors, but please in another pull request

File: src/main/java/com/epam/controller/ToursController.java [hunk 3]
```diff
@@ -79,28 +89,34 @@ public ModelAndView addTour(@RequestParam String tourType, @RequestParam String
                     .discountId(1) //stub
                     .build());
 
-            toursModel.addObject("result",result==1?"Success":"Failed to add");
+            toursModel.addObject("result", result == 1 ? "Success" : "Failed to add");
             return toursModel;
 
-        } catch (Exception e){
-            toursModel.addObject("result",e.getMessage());
+        } catch (Exception e) {
```
Comment (ghost) [inline src/main/java/com/epam/controller/ToursController.java:101]: can you log, that something went wrong?

File: src/main/webapp/WEB-INF/jsp/addtour.jsp [hunk 4]
```diff
@@ -1,153 +1,202 @@
...
         </div>
+
     </div>
+
     <div class="footer-copyright">
+
         <div class="container">
+
             © 2018 Copyright Text
+
```
Comment (ghost) [inline src/main/webapp/WEB-INF/jsp/addtour.jsp]: Oh my God, please see, what is commited.............

File: src/main/webapp/WEB-INF/jsp/hotels.jsp [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+                <ul>
+
+                    <li><a class="grey-text text-lighten-3" href="index"><spring:message code="homepage"/></a></li>
+
+                    <li><a class="grey-text text-lighten-3" href="#"><spring:message code="information"/></a></li>
+
+                    <li><a class="grey-text text-lighten-3" href="#"><spring:message code="feedback"/></a></li>
+
+                    <li><a class="grey-text text-lighten-3" href="#"><spring:message code="contacts"/></a></li>
+
```
Comment (ghost) [inline src/main/webapp/WEB-INF/jsp/hotels.jsp]: Why do have all these spaces???

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/controller/ToursController.java (hunk 3 above)
Comment (MaltsevaKsenia) [reply src/main/java/com/epam/controller/ToursController.java:101]: done

File: src/main/java/com/epam/controller/HotelController.java (hunk 1 above)
Comment (MaltsevaKsenia) [reply src/main/java/com/epam/controller/HotelController.java]: I think it is more like there were some extra elses. Please, look again, I've changed it

Comment (ghost) [reply src/main/java/com/epam/controller/HotelController.java]: okay, but please, replace it logic to the service layer and add tests for it

Comment (MaltsevaKsenia) [reply src/main/java/com/epam/controller/HotelController.java]: done

**Review by goponenkoev — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**